frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -0,0 +1,130 @@
+import type { MultiSelectProps, SelectItem } from "@mantine/core";
+import { MultiSelect } from "@mantine/core";
+import { useUncontrolled } from "@mantine/hooks";
+import type { ClipboardEvent, FocusEvent } from "react";
+import { useMemo, useState } from "react";
+
+export function MultiAutocomplete({
```

Comment:
I'll add separate unit tests in a follow-up PR. For now this is tested within `FilterValuePicker`.